yb-voyager/cmd/constants.go

```diff
@@ -65,13 +65,22 @@ const (
 	IMPORT_DATA_PHASE      = "import-data"
 	END_MIGRATION_PHASE    = "end-migration"
 	ASSESS_MIGRATION_PHASE = "assess-migration"
+	IMPORT_DATA_FILE_PHASE = "import-data-file"
 	//...more phases
-	OFFLINE              = "offline"
-	LIVE_MIGRATION       = "live migration"
+	OFFLINE        = "offline"
+	LIVE_MIGRATION = "live migration"
+	BULK_DATA_LOAD = "bulk data load from flat files"
+
+	AWS_S3      = "AWS-S3"
+	GCS_BUCKETS = "GCS-Buckets"
+	AZURE_BLOBS = "Azure-blob-storage"
+	LOCAL_DISK  = "Local-disk"
```

Comment:
Might be minor nit, but using mix and non-defined casing(upper/lower) might be confusing and hard to remember at user's end who is consuming this info, built an app based on it.

My approach is to use smaller cases wherever possible, unless its really required to use upper case.
Here everything i think can be lowercase, like `aws-s3`, gcs-buckets`, `local-disk`

Reply:
okay, I think only these four are the ones I kept mixed cases, will make it lower.